Renes_gimmicks/cards/Burning_rage.ts

```diff
@@ -0,0 +1,179 @@
+                "y": -50
+            },
+            "maxSpeed": 0,
+            "startRotation": {
+                "min": 265,
+                "max": 275
+            },
+            "noRotation": false,
+            "rotationSpeed": {
+                "min": 0,
+                "max": 0
+            },
+            "lifetime": {
+                "min": 1,
+                "max": 1.5
+            },
+            "blendMode": "normal",
+            "frequency": 0.45,
+            "emitterLifetime": -1,
+            "maxParticles": 20,
+            "pos": {
+                "x": 0,
+                "y": 0
+            },
+            "addAtBack": false,
+            "spawnType": "circle",
+            "spawnCircle": {
+                "x": 0,
+                "y": 0,
+                "r": 25
+            }
+        }, [texture]);
+    if (PixiUtils.containerUnits) {
+        const wrapped = Particles.wrappedEmitter(particleConfig, PixiUtils.containerUnits);
+        if (wrapped) {
+            const { container, emitter } = wrapped;
+            underworld.particleFollowers.push({
+                displayObject: container,
+                emitter,
+                target: follow
+            })
+        } else {
+            console.error('Failed to create BurnigRage particle emitter');
+        }
+    } else {
+        return;
+    }
+    //Particles.simpleEmitter(position, config, () => { }, Particles.containerParticlesUnderUnits);
+}
+
+const { refundLastSpell } = cards;
+const Unit = globalThis.SpellmasonsAPI.Unit;
+const { playDefaultSpellSFX } = cardUtils;
+const { CardCategory, probabilityMap, CardRarity } = commonTypes;
+
+const damageMultiplier = 8;
+const attackMultiplier =5;
+
+const cardId = 'Burning Rage';
+const spell: Spell = {
+    card: {
+        id: cardId,
+        category: CardCategory.Curses,
+        supportQuantity: true,
+        manaCost: 35,
+        healthCost: 0,
+        expenseScaling: 2,
+        probability: probabilityMap[CardRarity.RARE], 
+        thumbnail: 'spellmasons-mods/Renes_gimmicks/graphics/icons/Burninig_rage.png',
+        sfx: 'poison',
+        description: [`Each stack causes target to take ${damageMultiplier} damage, but also increases damage done by ${attackMultiplier}. Staks increase each turn`],
+        effect: async (state, card, quantity, underworld, prediction) => {
+            //Only filter unit thats are alive
+            const targets = state.targetedUnits.filter(u => u.alive);
+            //Refund if targets no one that can attack
+            if (targets.length == 0) {
+                refundLastSpell(state, prediction, 'No target, mana refunded')
+            } else {
+                if (!prediction){
+                    playDefaultSpellSFX(card, prediction);
+                }
+                for (let unit of targets) {
+                    Unit.addModifier(unit, card.id, underworld, prediction, quantity);
+                    unit.damage += quantity * attackMultiplier;
```

Comment:
Oh! It increases the unit's damage!  Very cool, I think the spell description could be a little more clear on that.  Maybe "but also increases the target's damage by ${attackMultiplier}"